internal/querycoordv2/balance/utils.go

```diff
@@ -0,0 +1,51 @@
+package balance
+
+import (
+	"context"
+	"time"
+
+	"github.com/milvus-io/milvus/internal/querycoordv2/task"
+)
+
+func CreateSegmentTasksFromPlans(ctx context.Context, checkerID int64, timeout time.Duration, plans []SegmentAssignPlan) []task.Task {
+	ret := make([]task.Task, 0)
+	for _, p := range plans {
+		actions := make([]task.Action, 0)
+		if p.To != -1 {
+			action := task.NewSegmentAction(p.To, task.ActionTypeGrow, p.Segment.GetID())
```

Comment:
Why use grow and reduce type instead of load and release, there are not intuitive